core/src/models/BibleReference.ts

```diff
@@ -87,6 +87,7 @@ export interface IBibleReferenceRangeQuery extends IBibleReferenceRange {
     versionUid: string;
     // TODO: implement
     targetVersionUid?: string;
+    skipPartialSectionsInDocument?: boolean;
```

Comment:
what is a partial section?

Reply:
... a section that starts at the beginning of the range but ends after it. e.g.: if you show Psalm 1:1 in a popup you usually only want to display the verse and not the multiple levels of section headers in the document. however in the full bible reader you want/need them all.